### ISSUE TITLE
QOL

### ISSUE THREAD

**Issue by xylix** [xylix/dotfiles#55]
various qol issues were here, changed to comment now

**Comment by xylix** [comment]
steal ideas from lazyvim / helix ( https://www.reddit.com/r/neovim/comments/16edwza/what_does_helix_get_right/ )

**Comment by xylix** [comment]
TODO: separate linux QOL to separate issue, mark as closed when not using linux

**Comment by xylix** [comment]
cut as non-relevant:
- Linux virtualbox
  - [ ] share VM's between users

- firefox
  - [ ] sync tabs continously

arch
- [ ] inactivity suspend
- [ ] disable yay / paru asking for cleanbuilds by default
- [ ] better solution than pacman -Qc or pacolog for seeing packages changes when updating https://stackoverflow.com/questions/5766666/how-to-view-release-notes-changelog-with-pacman

visuals
- [ ] good looking dunst themes
- [ ] applyy displaycalin profiilit
  - `z-displaycal-apply-profiles .desktop` 
  - https://wiki.archlinux.org/index.php/ICC_profiles#Colorimeter_or_spectrometer 
- [ ] system wide dark mode https://wiki.archlinux.org/index.php/Dark_mode_switching
- [ ] tty reso 
  - https://www.ondrovo.com/a/20160917-nvidia-tty-resolution/ 
  - https://www.reddit.com/r/archlinux/comments/4gwukx/nvidia_drivers_and_high_resolution_tty_possible/d2m3btk/
terminal:    
- [ ] visual scrollbar / ylös-alas position plonk elementti
- [ ] open textfiles in kitty nvim by default
  - https://wiki.archlinux.org/index.php/default_applications#xdg-open
  - xdg open kitty  
- vimwiki / logseq
  - [x] vim / vimwiki render logseq headers nicely
  - [ ] workign <CR> binds (check vimwiki/ftplugin/vimwiki.vim, add some lines to dotfiles/config/nvim/vimwiki.vim)
- linux desktop / i3
  - [ ] branching startup applications depending on username
  - [ ] some sort of X autostartup
    - https://wiki.archlinux.org/index.php/Xinit#Autostart_X_at_login 
  - [ ] brighten and dim only after X has been started 
    - https://superuser.com/questions/1037466/how-to-start-a-systemd-service-after-user-login-and-stop-it-before-user-logout  
    - https://superuser.com/questions/759759/writing-a-service-that-depends-on-xorg
    - https://www.google.com/search?client=firefox-b-d&q=systemctl+start+after+x11
  - [ ] HIDPI scaling on most applications
    - https://wiki.archlinux.org/index.php/HiDPI#Linux_console
    - https://askubuntu.com/questions/197828/how-to-find-and-change-the-screen-dpi
    - https://blog.summercat.com/configuring-mixed-dpi-monitors-with-xrandr.html
    - https://www.reddit.com/r/ManjaroLinux/comments/dgy8ll/wqhd_and_hdr_on_manjaro_thinkpad_x1c6/
    - [x] firefoxissa devPixelsperPx toimii https://winaero.com/enable-hidpi-scaling-firefox/
  - [ ] i3 number hotkeys to switch active windows within workspace (e.g. super 1...9 to switch workspace, alt 1...9 to switch window inside workspace)
  - https://www.reddit.com/r/i3wm/comments/hfodpv/ways_to_switch_windows_on_i3/
 - [ ] i3 resurrect save all workspaces https://github.com/JonnyHaystack/i3-resurrect
  - i3-save-tree varmaan preferable https://i3wm.org/docs/layout-saving.html#_saving_the_layout
  - [ ] cmd-c cmd-v copy paste https://superuser.com/questions/385748/binding-superc-superv-to-copy-and-paste
    - https://unix.stackexchange.com/questions/423264/no-more-keyboard-shortcut-possible-based-on-super-key-aka-windows-in-any-wm
    - Jos fishin alt+v keybind kilpailee se pitää disabloiga, g:"fish disable keybind"

- vim / neovim
  - [ ] autosave sessions with vim-obsession https://github.com/tpope/vim-obsession 
  - [ ] input lag benchmarking, plugin effect benchmarking
    - https://pavelfatin.com/typometer/
    - neovim profiler
